tests/test_qc_functions.py

```diff
@@ -1,16 +1,25 @@
 from __future__ import annotations
 
-import pytest  # noqa
+import itertools
+
+import numpy as np
+import numpy.ma as ma
+
+import _load_data
+import pytest
+from _settings import get_settings
+from cdm_reader_mapper import read_tables
+from cdm_reader_mapper.common.getting_files import load_file
```

Comment:
```suggestion
import pytest
```
I think, we don't need those imports since we directly want to test the qc routines here.